bit_manipulation/count_of_set_bits2.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ * @brief Finding No of Set Bits in O(k).
+ * https://www.geeksforgeeks.org/builtin-functions-gcc-compiler/
+ * @details
+ * How to calculate the no of Set bits in a number.
+ * math equations, etc.
+ * @author [Aman Deoli](https://github.com/deoliaman00)
+ * @see related_file.cpp, another_file.cpp
+ */
+
+#include <cassert>   /// for assert
+#include <stdio.h>   //stdio.h is a header file which has the necessary information to include the input/output related functions in our program.
+#include <iostream>
```

Comment:
Use `iostream` only. `stdio.h` is used only in C.
```suggestion
#include <iostream>    /// for IO operations
```